libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -320,6 +423,15 @@ async def search(
 
         Returns:
             list[Assistant]: A list of assistants.
+
```

Comment:
COuld we clarify how the m etadata filter works? Assuming it's an AND of exact match filters for each KV pair??